s30272-KamilKownacki/4/kolo_one.txt

```diff
@@ -33,7 +37,8 @@ $6b
 chmod a=- zasob
 $7
 ls|dd of=pliki.txt
-find . -type f | dd of=lista.txt
+find ./ -type f | dd of=lista.txt
+ls *.txt > listaplikow.txt

```

Comment:
Czy w zadaniu była mowa o listowaniu tylko obiektów z konkretnym rozszerzeniem? Nie. Więc to założenie jest błędne. Poza tym katalog także może mieć rozszerzenie "txt".